sdk/textanalytics/azure-ai-textanalytics/src/samples/java/com/azure/ai/textanalytics/TextAnalyticsClientJavaDocCodeSnippets.java

```diff
@@ -3,25 +3,1133 @@
 
 package com.azure.ai.textanalytics;
 
+import com.azure.ai.textanalytics.models.AnalyzeSentimentResult;
+import com.azure.ai.textanalytics.models.DetectLanguageInput;
+import com.azure.ai.textanalytics.models.DetectLanguageResult;
+import com.azure.ai.textanalytics.models.DetectedLanguage;
+import com.azure.ai.textanalytics.models.DocumentResultCollection;
+import com.azure.ai.textanalytics.models.ExtractKeyPhraseResult;
+import com.azure.ai.textanalytics.models.LinkedEntity;
+import com.azure.ai.textanalytics.models.NamedEntity;
+import com.azure.ai.textanalytics.models.RecognizeEntitiesResult;
+import com.azure.ai.textanalytics.models.RecognizeLinkedEntitiesResult;
+import com.azure.ai.textanalytics.models.RecognizePiiEntitiesResult;
+import com.azure.ai.textanalytics.models.TextAnalyticsRequestOptions;
+import com.azure.ai.textanalytics.models.TextDocumentBatchStatistics;
+import com.azure.ai.textanalytics.models.TextDocumentInput;
+import com.azure.ai.textanalytics.models.TextSentiment;
+import com.azure.core.http.HttpPipeline;
+import com.azure.core.http.HttpPipelineBuilder;
+import com.azure.core.util.Context;
+
+import java.util.Arrays;
+import java.util.List;
+
 /**
- * Code snippet for {@link TextAnalyticsClient}
+ * Code snippets for {@link TextAnalyticsClient} and {@link TextAnalyticsClientBuilder}
  */
 public class TextAnalyticsClientJavaDocCodeSnippets {
     private static final String SUBSCRIPTION_KEY = null;
     private static final String ENDPOINT = null;
+    private final TextAnalyticsClient textAnalyticsClient = new TextAnalyticsClientBuilder().buildClient();
+
+    /**
+     * Code snippet for creating a {@link TextAnalyticsClient} with pipeline
+     */
+    public void createAsyncTextAnalyticsClientWithPipeline() {
```

Comment:
Is this supposed to be in Async file?

Reply:
sorry. wrong method name, async should be removed